modules/frontend/manifests/packages.pp

```diff
@@ -8,14 +8,14 @@
   exec { "nodepackages":
     path    => "/bin:/usr/bin",
     unless  => "npm list -g --depth=0 | grep grunt-cli",
-    command => "npm install -g coffee-script bower grunt-cli gulp component yo",
```

Comment:
@RobLoach it's possible you might want to install things like the grunt-cli/bower globally. This should be OK for grunt-cli since the actual grunt libraries themselves are installed on the application level.https://github.com/gruntjs/grunt-cli

Less clear re bower. 

Also